README.md

```diff
@@ -8,60 +8,64 @@ Intended for [gettext](https://www.npmjs.com/search?q=gettext) or similar transl
 
 - Two functions
   - `gettext`
-    - Simple translation with language strng as lookupkey
+    - Simple translation with template string as translation lookup key
   - `ngettext`
-    - Singular/Plural aware translation with language strng as lookupkey
+    - Singular/Plural aware translation with template string as translation lookup key
 - Two ways of invoking
-  - simple values
-    - The argument/string is used as the lookup id
-  - object
-    - The key is used as the lookup id
+  - direct substitutions (simple values)
+    - Immediately substituted, same as normal template literals.
+    - Occurs **before** translation.
+    - Only permitted for primitives `string`, `number`, `boolean`, etc.
+  - object substitutions (single `key`/`value`)
+    - The `key` gives your translator context (and needs to survive translation).
+    - The `value` is substituted after **translation**.
 ### `gettext`
 
-Simple Values:
+- Without Substitution:
 
 ```js
 gettext`foo`
 ```
 
-key/val:
+- With Substitution:
 
 ```js
-gettext`Welcome to ${{location: 'hell'}}`
+gettext`Welcome to ${{location: 'Jurassic Park'}}`
 ```
 
-jsx (nested call):
+- jsx:
 
 ```js
 gettext`For more ${{link: <a href="http://google.com">{gettext`information`}</a>}}`
 ```
 
 ### `ngettext`
 
-- simple value:
+- Without Substitution:
 
 ```js
 ngettext(planets)`Hello world!|Hello worlds!`
 ```
 |   | input | output |
 |---|----|----|
-| **singular** | planets <= 1  | "Hello World!"  |
-| **plural**   | otherwise     | "Hello Worlds!" |
+| **singular** | `planets === 1`  | "Hello World!"  |
+| **plural**   | `planets > 1`    | "Hello Worlds!" |
```

Comment:
So this is determined by the underlying `ngettext` and I *think* you can have negative numbers. Certainly that is how the `ngettextDefault` is implemented. In that case `otherwise` is more correct that what you have now.

Obviously we are just talking about `EN` locale. An interesting [table](http://docs.translatehouse.org/projects/localization-guide/en/latest/l10n/pluralforms.html) shows some others.

Reply:
ok - i just ran it and i see what you mean about anything other than 1 being regarded as plural